feflow/tests/test_lambdaprotocol.py

```diff
@@ -0,0 +1,52 @@
+import os
+import pytest
+
+from feflow.utils import lambda_protocol
+
+running_on_github_actions = os.environ.get('GITHUB_ACTIONS', None) == 'true'
+
+
+def test_lambda_protocol():
+    """
+
+    Tests LambdaProtocol, ensures that it can be instantiated with defaults, and that it fails if disallowed functions are tried
+
+    """
+
+    # check that it's possible to instantiate a LambdaProtocol for all the default types
+    for protocol in ['default', 'namd', 'quarters']:
+        lp = lambda_protocol.LambdaProtocol(functions=protocol)
+        assert isinstance(lp, lambda_protocol.LambdaProtocol), "instantiated is not instance of LambdaProtocol."
+
+
+
+"""this test is a little unhappy
+
+it checks that missing terms are added back in
+
+however a more recent commit in openfe land changed this to error not warn
+
+so the test as-is can't function
```

Comment:
If not already done, could you raise an issue about this? It would be good to make sure we don't forget to fix this.